fsharp-backend/src/LibExecution/StdLib/LibChar.fs

```diff
@@ -19,41 +19,37 @@ let incorrectArgs = LibExecution.Errors.incorrectArgs
 let varA = TVariable "a"
 let varB = TVariable "b"
 
-let fns : List<BuiltInFn> = []
-  // [ { name = fn "Char" "toASCIICode" 0
-  //
-  //   ; parameters = [Param.make "c" TCharacter ""]
-  //   ; returnType = TInt
-  //   ; description = "Return `c`'s ASCII code"
-  //   ; fn =
-  //        (fun _ -> Exception.code "This function no longer exists.")
-  //   ; sqlSpec = NotYetImplementedTODO
-  //   ; previewable = Pure
-  //   ; deprecated = ReplacedBy(fn "" "" 0) }
-  // ; { name = fn "Char" "toASCIIChar" 0
-  //   ; parameters = [Param.make "i" TInt ""]
-  //   ; returnType = TCharacter
-  //   ; description = "convert an int to an ASCII character"
-  //   ; fn =
-  //        (fun _ -> Exception.code "This function no longer exists.")
-  //   ; sqlSpec = NotYetImplementedTODO
-  //   ; previewable = Pure
-  //   ; deprecated = ReplacedBy(fn "" "" 0) }
-  // ; { name = fn "Char" "toLowercase" 0
-  //   ; parameters = [Param.make "c" TCharacter ""]
-  //   ; returnType = TCharacter
-  //   ; description = "Return the lowercase value of `c`"
-  //   ; fn =
-  //        (fun _ -> Exception.code "This function no longer exists.")
-  //   ; sqlSpec = NotYetImplementedTODO
-  //   ; previewable = Pure
-  //   ; deprecated = ReplacedBy(fn "" "" 0) }
-  // ; { name = fn "Char" "toUppercase" 0
-  //   ; parameters = [Param.make "c" TCharacter ""]
-  //   ; returnType = TCharacter
-  //   ; description = "Return the uppercase value of `c`"
-  //   ; fn =
-  //        (fun _ -> Exception.code "This function no longer exists.")
-  //   ; sqlSpec = NotYetImplementedTODO
-  //   ; previewable = Pure
-  //   ; deprecated = ReplacedBy(fn "" "" 0) } ]
+let fns : List<BuiltInFn> = 
+  [ { name = fn "Char" "toASCIICode" 0
+      parameters = [Param.make "c" TChar ""]
+      returnType = TInt
+      description = "Return `c`'s ASCII code"
+      fn = Errors.removedFunction
+      sqlSpec = NotQueryable
+      previewable = Pure
+      deprecated = DeprecatedBecause("Dark does not support Characters yet") }
```

Comment:
Good guess, but not exactly right. Dark does actually support characters (just not character literals, you have to use `String::foreach` to get them I think), but it's that these functions supported an old ascii-based char, and these days chars are EGC-based. So deprecated because they "used an old Character type that no longer exists"